docs/StartHere.md

```diff
@@ -0,0 +1,9 @@
+# Tidepool uploader developer guide
+
+As you're getting ready to develop code in this repository, we recommend starting with the following documents:
```

Comment:
Are these headings based on @darinkrauss 's suggested "template", or has it been identified as a specific need? I guess what I'm asking is - can they be chopped and changed or are they set in stone?